### PR TITLE
Add test for FactTable.getFactByKey

### PR DESCRIPTION
**Pull request by lacinoire**
Hey 😊
I want to contribute the following test:

Test that `table.getFactByKey("Iwcz&T&5+;N4Sb)kE+#P")` is null when `getFactByKey` is called with the parameter `key = "Iwcz&T&5+;N4Sb)kE+#P"`.
This tests the method [`FactTable.getFactByKey`](https://github.com/synthetichealth/synthea/blob/e5d35d2a9f9cb7193e797bfc292e5f782e436d29/src/main/java/org/mitre/synthea/helpers/FactTable.java#L75).
This test is based on the test [`testFactTable`](https://github.com/synthetichealth/synthea/blob/e5d35d2a9f9cb7193e797bfc292e5f782e436d29/src/test/java/org/mitre/synthea/helpers/FactTableTest.java#L12).

Curious to hear what you think!

(I wrote this test as part of a research study at TU Delft. [Find out more](https://github.com/lacinoire/lacinoire/blob/main/README.md))

### CODE REVIEW

File: src/test/java/org/mitre/synthea/helpers/FactTableTest.java [hunk 1]
```diff
@@ -37,4 +37,14 @@ public void testFactTable() {
...
 
+  @Test
+  public void testGetFactByKey() {
+    // Setup a basic fact table with a header.
+    FactTable table = new FactTable();
+    table.setHeader("ID,KEY,NAME,DESCRIPTION");
+    // Insert some facts.
+    int h = table.addFact("H", "Hydrogen,Highly flammable gas");
+    Assert.assertNull(table.getFactByKey("Iwcz&T&5+;N4Sb)kE+#P"));
+  }
```
Comment (jawalonoski) [inline src/test/java/org/mitre/synthea/helpers/FactTableTest.java:48]:
```suggestion
    Assert.assertEquals("Hydrogen,Highly flammable gas", table.getFactByKey("H"));
  }
```

This will add a positive test as well as the negative test. You could break them out into separate unit tests if you wanted.

**Review by jawalonoski — APPROVED** [review]
Thank you for the pull request. We will gladly take more unit tests!
